app/controllers/two_factor_authentication/webauthn_verification_controller.rb

```diff
@@ -138,5 +141,13 @@ def check_sp_required_mfa
     def platform_authenticator?
       params[:platform].to_s == 'true'
     end
+
+    def webauthn_configuration_or_latest
+      form&.webauthn_configuration || webauthn_configurations.take
+    end
+
+    def webauthn_configurations
+      MfaContext.new(current_user).webauthn_configurations.order(:created_at)
```

Comment:
I think this will default to ascending order

Reply:
Ah, good catch! I think I assumed `take` worked like a "pop", but upon closer inspection it doesn't have a guaranteed order. I'll fix this up and add test coverage.

Reply:
Should be fixed with 7be695f, but test coverage TBD.

Reply:
This discussion of sort order had me wondering if it has an impact on how we're using `#credentials` to pass through to the relevant WebAuthn APIs, and apparently they do! I think I'll change this to descending order, since it may be reasonable to expect that the latest-created authenticator is the most preferred one.

>[...] a list of [PublicKeyCredentialDescriptor](https://www.w3.org/TR/webauthn-2/#dictdef-publickeycredentialdescriptor) objects representing [public key credentials](https://www.w3.org/TR/webauthn-2/#public-key-credential) acceptable to the caller, in descending order of the caller’s preference (the first item in the list is the most preferred credential, and so on down the list).

https://www.w3.org/TR/webauthn-2/#dom-publickeycredentialrequestoptions-allowcredentials

Reply:
Changed sort order and improved spec coverage in ccb5697.